tools/serverpod_cli/lib/src/logger/logger.dart

```diff
@@ -0,0 +1,35 @@
+/// Serverpods internal logger interface.
+/// All logging output should go through this interface.
+/// The purpose is to simplify implementing and switching out concrete logger
+/// implementations.
+abstract class Logger {
+  /// Display a [message] wrapped in a box.
+  /// Commands should use this when they want to highlight a message that otherwise is easily missed.
+  void printBox(String message);
```

Comment:
This feels misplaced.

This class more communicates the log level of the message while this method communicates a style. And error might want to have a box as well.

Should we remove this and add it as a style option for Info for now.

Reply:
A style option sounds like a better choice. I think we should have the same style options for all log levels.